names/names.py

```diff
@@ -1,5 +1,21 @@
 import time
 
+def quicksort_out_of_place(arr):
```

Comment:
Like I said in 1:1, original idea is that you'd use a BST here, fill it with the names from names1 and then run `bst.contains(name)` for each name in names2. But I do like this solution; I think it's great that you're thinking about using tools from all of CS so far, not just the past week